src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -701,11 +697,7 @@ private static async
                         ArrayPool<byte>.Shared.Return(toReturn, clearArray: true);
                     }
 
-                    lastRead = await stream.ReadAsync(
-                        rented,
-                        written,
-                        rented.Length - written,
-                        cancellationToken).ConfigureAwait(false);
+                    lastRead = await stream.ReadAsync(rented.AsMemory(written, rented.Length - written), cancellationToken).ConfigureAwait(false);
```

Comment:
Here as well.

Reply:
Sure, no problem. BTW, this is a case that can be simplified as Levi pointed out above: `rented.AsMemory(written)`.